org_fedora_oscap/content_handling.py

```diff
@@ -50,6 +51,47 @@ class ContentCheckError(ContentHandlingError):
 
     pass
 
+class ParseHTMLContent(HTMLParser):
+    """Parser class for HTML tags within content"""
+
+    def __init__(self):
+        HTMLParser.__init__(self)
+        self.content = ""
+
+    def handle_starttag(self, tag, attrs):
+        if tag == "html:ul":
+            self.content = self.content + "\n"
+        elif tag == "html:li":
+            self.content = self.content + "\n"
+        elif tag == "html:br":
+            self.content = self.content + "\n"
```

Comment:
Parser goes through the content fed, and acts on each start and end of a tag, and also on content within these tags.

The added new lines are to be on the safer side of readability of content, because `xmllint` messes with human readability in a incomprehensible way to me. Take for example description of `rhev-upstream` Profile in built datastream:

```
        <ns10:description override="true" xml:lang="en-US">This is a *draft* profile for STIG. This profile is being developed under the DoD consensus model to become a STIG in coordination with DISA FSO.<html:br/><html:br/>

<html:p><html:strong>Where is the RHV-H STIG?</html:strong><html:ul><html:li>Question: May I deploy a product if no STIG exists?<html:br/>
            Answer: Yes, based on mission need and with DAA approval.
        </html:li><html:li>Question: What do I use if there is no STIG?<html:br/>
            Answer: DISA FSO developed Security Requirement Guides (SRGs) to address technology areas. In the absence of a STIG, an SRG can be used to determine compliance with DoD policies. If there is no applicable SRG or STIG, industry or vendor recommended practices may be used. Examples include Center for Internet Security Benchmarks, Payment Card Industry requirements or the vendor's own security documentation.
        </html:li></html:ul><html:small>Source: http://iase.disa.mil/stigs/Pages/faqs.aspx#STIG</html:small></html:p>
</ns10:description>
```
This was nicely formatted in the source: https://github.com/OpenSCAP/scap-security-guide/blob/master/RHEL/7/input/profiles/stig-rhvh-upstream.xml

Without double newlines, the description output is like below, the Source reference shows it self attached to last list item:
```
This is a *draft* profile for STIG. This profile is being developed under the DoD consensus model to become a STIG in coordination with DISA FSO.

Where is the RHV-H STIG?

Question: May I deploy a product if no STIG exists?
Answer: Yes, based on mission need and with DAA approval.
Question: What do I use if there is no STIG?
Answer: DISA FSO developed Security Requirement Guides (SRGs) to address technology areas. In the absence of a STIG, an SRG can be used to determine compliance with DoD policies. If there is no applicable SRG or STIG, industry or vendor recommended practices may be used. Examples include Center for Internet Security Benchmarks, Payment Card Industry requirements or the vendor's own security documentation.Source: http://iase.disa.mil/stigs/Pages/faqs.aspx#STIG
```